boards/arm/nucleo_l552ze_q/doc/nucleol552ze_q.rst

```diff
@@ -276,7 +276,16 @@ You should see the following message on the console:
 Building a secure/non-secure with Arm |reg| TrustZone |reg|
 -----------------------------------------------------------
 
-The TF-M integration sample :ref:`tfm_ipc` can be run by a Nucleo L552ZE Q, using the ``nucleo_l552ze_q_ns`` target. When building a ``*_ns`` image with TF-M, a ``build/tfm/install/postbuild.sh`` bash script will be run as a post-build step to make some required flash layout changes. The ``build/tfm/install/postbuild.sh`` script will also be used to flash the board. Check the ``build/tfm/install`` directory to ensure that the commands required by these scripts (``readlink``, etc.) are available on your system.
+The TF-M integration sample :ref:`tfm_ipc` can be run by a Nucleo L552ZE Q,
+using the ``nucleo_l552ze_q_ns`` target. When building a ``*_ns`` image with TF-M,
+a ``build/tfm/install/postbuild.sh`` bash script will be run as a post-build step
+to make some required flash layout changes. The ``build/tfm/regression.sh`` script
+will need to be run to perform device initialization, and then run ``west flash --hex-file build/tfm_merged.hex``
```

Comment:
Can you review the command here?
I had to go in `samples/tfm_integration/tfm_ipc/build` and then launch `west flash --hex-file tfm_merged.hex`

Reply:
Yes, this needs an improvement.

Reply:
Looks like the build output is generated to the sample project directory instead of the root directory. I did a clean run and can see the merged hex file is generated at `build/tfm_merged.hex`. Do you have any local environments that may impact?

Reply:
@yestin the file is present here indeed, though, when running the command, I'm getting the following west error:
```
$ west flash --hex-file build/stm32l562e_dk_ns/tfm_merged.hex
FATAL ERROR: --build-dir was not given, /local/mcu/zephyrproject/zephyr/samples/tfm_integration/tfm_ipc is not a build directory and the default build directory cannot be determined. Check your build.dir-fmt configuration option
```
@ioannisg is that what you're seeing as well ?

Reply:
@yestin I confirm this is only due to my local build.dir-fmt configuration.
I've raised https://github.com/zephyrproject-rtos/zephyr/issues/33498 to discuss it